examples/examples_test.go

```diff
@@ -44,6 +44,7 @@ import (
 	"k8s.io/kubernetes/pkg/registry/batch/job"
 	schedulerapi "k8s.io/kubernetes/pkg/scheduler/api"
 	schedulerapilatest "k8s.io/kubernetes/pkg/scheduler/api/latest"
+	apivalidation "k8s.io/kubernetes/pkg/scheduler/api/validation"
```

Comment:
nit: Please rename to `schedulerapivalidation`

Reply:
OK,I have done it,Thanks

Reply:
I don't see this change. Did you forget to push your new changes?

Reply:
The change was there yesterday, but it has gone back to the first version of the PR.

Reply:
cc @brendandburns @bsalamat  I have re pushed it now,Thanks